airbyte-db/db-lib/src/main/java/io/airbyte/db/jdbc/JdbcUtils.java

```diff
@@ -62,4 +65,8 @@ public static Map<String, String> parseJdbcParameters(final String jdbcPropertie
     return parameters;
   }
 
+  public static boolean useSsl(final JsonNode config) {
+    return !config.has(SSL_KEY) || config.get(SSL_KEY).asBoolean();
```

Comment:
To be quite frank, I'm not completely sure the reason for why `useSsl` would want to return `true` if there is no `SSL_KEY`. I mainly refactored this portion of code from other parts of the code base

However, it seem that all the places I saw this usage of `useSsl` if no `SSL_KEY` has been set in the config object that we want to set `ssl` in the config. My presumption would be because these connections require `ssl` to operate correctly. I can update the function with javadoc comment that the code checks for if no `SSL_KEY` exists we also return `true` so it's clear in case there's a misunderstanding on the functionality of this method. Would there be a group or individual that knows more about why this method operates the way it does?

Reply:
I was able to clarify the logic and will be adding some tests to codify the logic but the `asBoolean` is the key to understanding this method. The `asBoolean` function will map a value to a boolean if it can (e.g. `<int>0 -> <boolean>false`, `<string>false -> <boolean>false`, `<int>3 -> <boolean> true`). Basically the logic is: 

`If SSL_KEY is not set OR (if SSL_KEY is set AND the value of the SSL_KEY is mapped to true) THEN we set ssl fields`